softserve-orm/src/main/java/com/softserve/task4/orm/classes/Main.java

```diff
@@ -0,0 +1,33 @@
+package com.softserve.task4.orm.classes;
+
+import com.softserve.task4.orm.client.MySQLConnector;
+import com.softserve.task4.orm.processes.InsertColumns;
+
+import java.sql.Connection;
+
+/**
+ * Created by ${JDEEK} on ${11.11.2018}.
+ */
+public class Main {
+    public static void main(String[] args){
+        try {
+            MySQLConnector client = new MySQLConnector("jdbc:mysql://localhost/test?serverTimezone=Europe/Moscow&useSSL=false",
```

Comment:
Are you sure that main() - is the best place for this code? )
Please move a creating a connection from main to some config file in the next request.